### ISSUE TITLE
Filter popup in the grid not removed from DOM

### ISSUE THREAD

**Issue by andreavitali** [telerik/kendo-angular#1529]
I got a problem with menu filters popup in the grid.

## I'm submitting a...
* Bug report 

## Current behavior
Filter popup is closed correctly only clicking on the 'funnel' icon. If I click outside the popup the <kendo-popup> element remains in the DOM without content but, due to its borders, an "artifact" remains on the grid.

## Expected behavior
The <kendo-popup> element should be removed from DOM as it happens clicking on the filter icon.

## Minimal reproduction of the problem with instructions
I wasn't able to reproduce the error in a plunkr, but i can show an image of the artifact i was talking about:

![popup errir](https://user-images.githubusercontent.com/6029554/39916931-45bf0e1a-550c-11e8-87a5-613d43668d59.png)

This happened after i opened the Status column filter and click outside the popup.
I suppose the gray line is the result of this part of .k-popup CSS:
`
border-width: 1px; 
border-style: solid;
`
I got the same problem using kendo-theme-bootstrap and kendo-theme-default, but i don't think it's a style problem. What could be the cause that stop the popup from being removed from the DOM?

## Environment

Package versions:
```
    "@angular/animations": "^5.2.10",
    "@angular/common": "^5.2.10",
    "@angular/compiler": "^5.2.10",
    "@angular/core": "^5.2.10",
    "@angular/forms": "^5.2.10",
    "@angular/http": "^5.2.10",
    "@angular/platform-browser": "^5.2.10",
    "@angular/platform-browser-dynamic": "^5.2.10",
    "@angular/router": "^5.2.10",
    "@progress/kendo-angular-buttons": "^4.0.0",
    "@progress/kendo-angular-dateinputs": "^3.0.0",
    "@progress/kendo-angular-dialog": "^3.4.0",
    "@progress/kendo-angular-dropdowns": "^2.1.0",
    "@progress/kendo-angular-excel-export": "^1.0.7",
    "@progress/kendo-angular-grid": "^3.2.0",
    "@progress/kendo-angular-inputs": "^3.0.0",
    "@progress/kendo-angular-intl": "^1.4.0",
    "@progress/kendo-angular-l10n": "^1.1.0",
    "@progress/kendo-angular-layout": "^2.2.1",
    "@progress/kendo-data-query": "^1.2.2",
    "@progress/kendo-drawing": "^1.5.1",
    "@progress/kendo-theme-bootstrap": "^2.11.11",
    "bootstrap": "^4.1.1",
    "core-js": "^2.5.3",
    "font-awesome": "^4.7.0",
    "moment": "^2.22.1",
    "rxjs": "^5.5.10",
    "zone.js": "^0.8.26"
```

Browser:
- Chrome (desktop) version 66
- Edge version 41

**Comment by danielkaradachki** [comment]
Could you try updating the `@progress/kendo-angular-popup` package? The issue that caused the problem with the grid filter should be fixed in [2.2.0](https://www.telerik.com/kendo-angular-ui/components/popup/changelog/#toc-bug-fixes)

**Comment by andreavitali** [comment]
Thank you, now it's working.
I didn't have `@progress/kendo-angular-popup` in `package.json` so i thought it was updated together with `@progress/kendo-angular-grid`, that's why i didn't read the popup component changelog.
